content/posts/2025/02/core-team-promotions.md

```diff
@@ -1,40 +1,32 @@
 ---
 layout: page
-authors: ["SherAaron Hurt", "Danielle Sieh", “Angelique Trusler”]
+authors: ["SherAaron Hurt", "Danielle Sieh", "Angelique Trusler"]
 teaser: "Introducing our newest Junior Team Lead and Senior Program Manager"
 title: "Carpentries Core Team Promotions: Congratulations Angelique and Danielle!"
-date: 2025-02-10
+date: 2025-02-11
 time: "09:00:00"
 tags: ["Core Team", "Workshops", "Community Engagement"]
 ---
 
-
-# A Message From SherAaron Hurt
-
+## A Message From SherAaron Hurt
 Cheerfully, I get to share some exciting news! Two of our outstanding and well-deserving Core Team members have advanced in their roles at The Carpentries, Inc.
 
-
-## Danielle Sieh, Workshops & Training Manager
-
+### Danielle Sieh, Workshops & Training Manager
 Danielle Sieh, who joined The Carpentries in [February 2021](https://carpentries.org/blog/2021/02/welcoming-danielle/) as a Workshop Administrator, has been promoted to Senior Program Manager for the Workshops and Instruction Team. Over the past year as Junior Program Manager, Danielle has showcased exceptional leadership within the Instructor community, demonstrated outstanding organisational skills in shaping content for our new website and Handbooks, and successfully managed the Self-Organised Workshops project.
 In her new role, Danielle will take on additional leadership responsibilities, focusing on planning and process improvement within her areas of oversight. 
  
-
-## Dr. Angelique Trusler, Associate Director of Community 
-
+### Dr. Angelique Trusler, Associate Director of Community 
 Dr. Angelique Trusler began her journey with The Carpentries in [November 2019](https://carpentries.org/blog/2019/11/introducing-angelique/) as the Regional Consultant for Southern Africa. Since then, she has also served as African Capacity Development Manager and Community Manager. Now, she steps into the role of Associate Director of Community, a Junior Team Lead position.

```

Comment:
```suggestion
Dr Angelique Trusler began her journey with The Carpentries in [November 2019](https://carpentries.org/blog/2019/11/introducing-angelique/) as the Regional Consultant for Southern Africa. Since then, she has also served as African Capacity Development Manager and Community Manager. Now, she steps into the role of Associate Director of Community, a Junior Team Lead position.
```